README.md

```diff
@@ -230,6 +254,29 @@ Example:
 }
 ```
 
+### Accessing the system
+
+With a virtual machine launched with the above [virt-install](#virt-install) example, access the system with
+
+```
+ssh -i /path/to/private/ssh-key foo@ip-address
```

Comment:
We have two example configs, one with username `foo` for the general build config example and one with `alice` for the user specific example.  I doubt anyone will be confused by this but maybe we should have just one username.

Reply:
let's go with alice! i like that better :) i pushed the update...